app/src/main/res/values-ru/strings.xml

```diff
@@ -0,0 +1,218 @@
+    <string name="menu_item_title_delete_favorite">Удалить</string>
+    <string name="menu_item_title_set_alarm">Задать напоминание</string>
+    <string name="menu_item_title_delete_alarm">Очистить напоминание</string>
+    <string name="menu_item_title_delete_all">Удалить всё</string>
+    <string name="menu_item_title_favorites">Избранное</string>
+    <string name="menu_item_title_feedback">Обратная связь</string>
+    <string name="menu_item_title_navigate">Навигация</string>
+    <string name="menu_item_title_refresh">Обновить</string>
+    <string name="menu_item_title_share_event">Поделиться мероприятием</string>
+    <string name="menu_item_title_share_favorites">Поделиться избранным</string>
+
+    <!-- AlertDialogHelper -->
+    <string name="OK">ОК</string>
+
+    <!-- CertificateDialog -->
+    <string name="dlg_certificate_message_fmt">
+        Не удалось установить защищенное соединение.\n
+        (<xliff:g id="error" example="Unknown Error">%s</xliff:g>)
+    </string>
+    <string name="dlg_invalid_certificate_title">Доверять неизвестному сертификату?</string>
+    <string name="dlg_invalid_certificate_could_not_apply">Невозможно добавить сертификат.</string>
+
+    <!-- CustomHttpClient -->
+    <string name="dlg_err_connection_failed">Ошибка подключения</string>
+    <string name="dlg_err_failed_unknown_host">Неизвестный адрес:\n%s</string>
+    <string name="dlg_err_failed_wrong_http_credentials">Неверная HTTP-аутентификация</string>
+    <string name="dlg_err_failed_timeout">Сервер не отвечает (таймаут).</string>
+    <string name="dlg_err_failed_connect_failure">Неизвестная ошибка подключения</string>
+    <string name="dlg_err_failed_not_found">Неверный URL расписания</string>
+    <string name="dlg_err_failed_parse_failure">Не удалось разобрать ответ</string>
+    <string name="dlg_err_failed_ssl_failure">Ошибка SSL</string>
+    <string name="dlg_err_failed_http_cleartext_not_permitted">HTTP запрещен. Пожалуйста, используйте HTTPS.</string>
+    <string name="uptodate">Расписание обновлено</string>
+
+    <!-- Schedule changes dialog -->
+    <string name="schedule_changes_dialog_title">Обновление расписания</string>
+    <string name="schedule_changes_dialog_updated_to_text">Расписание обновлено до\u0020</string>
+    <string name="schedule_changes_dialog_changed_new_cancelled_text">. Нет изменений в <xliff:g id="num_changes" example="3 lectures">%1$s</xliff:g>, <xliff:g id="num_new" example="2 are">%2$s</xliff:g> новые, <xliff:g id="num_canceled" example="1 is">%3$s</xliff:g> отменены.</string>
+    <string name="schedule_changes_dialog_affected_text">Изменения затронули <xliff:g id="num_marked" example="2">%1$d</xliff:g> из вашего списка избранных.</string>
+    <plurals name="schedule_changes_dialog_number_of_lectures">
+        <item quantity="one">%d лекция</item>
+        <item quantity="few">%d лекции</item>
+        <item quantity="other">%d лекций</item>
+    </plurals>
```

Comment:
The lint check in Android Studio tells that for `schedule_changes_dialog_number_of_lectures` also `many` should be defined. Can you tell if the warning is correct and if we need another item here because of a special plural word?